src/OrbitBase/ThreadUtilsLinux.cpp

```diff
@@ -40,12 +40,15 @@ std::string GetThreadName(pid_t tid) {
   return comm_content.value();
 }
 
-void SetCurrentThreadName(const std::string& thread_name) {
-  std::string name = thread_name;
-  if (name.length() >= kMaxThreadNameLength) {
-    name[kMaxThreadNameLength - 1] = '\0';
+void SetCurrentThreadName(std::string_view thread_name) {
+  std::string buf;
+  const char* thread_name_cstr = thread_name.data();
```

Comment:
thread_name_cstr won't be null terminated which is expected by pthread_setname_np expects

Reply:
right. fixed.